### ISSUE TITLE
L2 normalization for hid_resp and hid_cont

### ISSUE THREAD

**Issue by mudong0419** [jasonwu0731/ToD-BERT#18]
@jasonwu0731 Thanks for your excellent work. 
 Calculate RCL loss
  scores = torch.matmul(hid_cont, hid_resp.transpose(1, 0))
  loss_rs = xeloss(scores, resp_label)
  loss += loss_rs
  loss_rs = loss_rs.item()

when calculate rs loss, is it nessary to L2 normalize hid_resp hid_cont before matmul? So that the [CLS] vector be an embedding of a sentence.

**Comment by jasonwu0731** [comment]
Hi, 

I did not get what you are suggesting, can you point me to the code or equation? Thank you
